.mailmap

```diff
@@ -2,10 +2,12 @@ Andy Jack <ajack@maxmind.com> <andyjack@users.noreply.github.com>
 Andy Jack <ajack@maxmind.com> <github@veracity.ca>
 Dave Rolsky <drolsky@maxmind.com> <autarch@urth.org>
 Greg Oschwald <goschwald@maxmind.com> <oschwald@gmail.com>
-Greg Oschwald <goschwald@maxmind.com> Gregory Oschwald <goschwald@maxmind.com>
-Mateu Hunter <mhunter@maxmind.com> <hunter@missoula.org>
+Greg Oschwald <goschwald@maxmind.com> <goschwald@maxmind.com>
```

Comment:
Does this say that anything committed by Greg's MaxMind email address regardless of name is mapped to the canonical "Greg Oschwald" at his MaxMind email address?

Reply:
Yeah. You can see the difference by running `git shortlog -e` before and after the changes.